lale/operators.py

```diff
@@ -1357,6 +1362,13 @@ def __getattr__(self, name: str) -> Any:
                 return "classifier"  # satisfy sklearn.base.is_classifier(op)
             elif self.is_regressor():
                 return "regressor"  # satisfy sklearn.base.is_regressor(op)
+        if name == "classes_":
+            if hasattr(self.impl, "estimator") and hasattr(
```

Comment:
Also, would we support "mitigator" also?  if so, maybe we should use `_final_estimator` here so that we reuse the same logic